docs/en/latest/building-apisix.md

```diff
@@ -276,7 +276,7 @@ For the error `Error unknown directive "lua_package_path" in /API_ASPIX/apisix/t
 To run a specific test case, use the command below:
 
 ```shell
-prove -Itest-nginx/lib -r t/plugin/openid-connect.t
+make test test=t/plugin/openid-connect.t
```

Comment:
IMO, I am used to using

```
prove -I../test-nginx/lib t/plugin/openid-connect.t
```

This way does not rely on Makefile and is suitable for developing various `lua-resty-*` repositories.

Reply:
5. Run the tests by running:

   ```shell
   make test
   ```
Consistent with the above command, more convenient to remember

Reply:
If there are other test commands, they can all be standardized into this form

Reply:
New developers don't need to know about test-nginx, they just need to remember `make test`

Reply:
It's for new developers, we could use `prove` anyway

Reply:
`make test test=xxx` maybe confusion? what about `make test case=xxx`?

Reply:
i agree with `make test case=xxx`,  what about @soulbird ?

Reply:
+1.